app/api/v1/entities/shallow.rb

```diff
@@ -35,6 +39,7 @@ class ProjectRoleShallow < Shallow
 
   class ProtocolShallow < Shallow
     root 'protocols', 'protocol'
+    expose :short_title
```

Comment:
It's a tradeoff because if we don't expose :short_title in Shallow then we're making a separate web service call for Full protocols and will receive 29 other columns we don't need, multiplied by the number of requested protocols. With that said, we can update our web service client to make a separate call for full protocols and wait until performance becomes an issue. How should we proceed: as currently coded or remove short_title?

Also, I think IdentityShallow should expose :email, :first_name, :last_name, and :ldap_uid because it seems like every scenario in which I'll be looking up an Identity, I'll be doing it so I can display and/or use all of those fields. What do you think?